can/interfaces/socketcan/socketcan.py

```diff
@@ -555,6 +556,7 @@ def capture_message(
         is_remote_frame=is_remote_transmission_request,
         is_error_frame=is_error_frame,
         is_fd=is_fd,
+        is_rx=not is_tx,
```

Comment:
I think `is_rx` should follow the same pattern as the other parameters, i.e. `is_rx=is_rx`. So move the `not` to L539 instead.

Reply:
Done